content/v2.0/write-data/sample-data/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Sample data
+description: >
+  Use sample data to familiarize yourself with time series data and InfluxDB.
+  InfluxData provides many sample time series datasets to use with InfluxDB
+  and InfluxDB Cloud.
+menu:
+  v2_0:
+    parent: Write data
+weight: 105
+---
+
+Use sample data to familiarize yourself with time series data and InfluxDB.
+InfluxData provides many sample time series datasets to use with **InfluxDB** and **InfluxDB Cloud**.
+
+{{< children >}}
+
+---
+
+### Air sensor sample data
```

Comment:
I don't think we're using these demo data sets anymore :( just website monitoring using telegraf to ping a number of urls

Reply:
We still provide and use sample data sets in the documentation and a few other places.

Reply:
ah ok ! :)